content/ko/docs/concepts/extend-kubernetes/extend-cluster.md

```diff
@@ -0,0 +1,205 @@
+---
+title: 쿠버네티스 클러스터 확장
+content_template: templates/concept
+weight: 10
+---
+
+{{% capture overview %}}
+
+쿠버네티스는 고도로 구성 가능하고 확장 가능하다. 결과적으로
+쿠버네티스 프로젝트 코드에 패치를 포크하거나 제출할 필요가
+거의 없다.
+
+이 가이드는 쿠버네티스 클러스터를 사용자 정의하기 위한 옵션을 설명한다.
+쿠버네티스 클러스터를 업무 환경의 요구에 맞게
+조정하는 방법을 이해하려는 {{< glossary_tooltip text="클러스터 운영자" term_id="cluster-operator" >}}를 대상으로 한다.
+잠재적인 {{< glossary_tooltip text="플랫폼 개발자" term_id="platform-developer" >}} 또는 쿠버네티스 프로젝트 {{< glossary_tooltip text="컨트리뷰터" term_id="contributor" >}}인 개발자도
+익스텐션 포인트와 패턴이 무엇인지,
+그리고 장단점에 대한 소개 자료로 유용한 내용을 찾을 수 있을 것이다.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## 개요
+
+사용자 정의 방식은 크게 플래그, 로컬 구성 파일 또는 API 리소스 변경만 포함하는 *구성* 과 추가 프로그램이나 서비스 실행과 관련된 *익스텐션* 으로 나눌 수 있다. 이 문서는 주로 익스텐션에 관한 것이다.
+
+## 구성
+
+*구성 파일* 및 *플래그* 는 각 명령에 대한 온라인 문서의 참조 섹션에 문서화되어 있다.
+
+* [kubelet](/docs/admin/kubelet/)
+* [kube-apiserver](/docs/admin/kube-apiserver/)
+* [kube-controller-manager](/docs/admin/kube-controller-manager/)
+* [kube-scheduler](/docs/admin/kube-scheduler/).
+
+호스팅된 쿠버네티스 서비스 또는 매니지드 설치 환경의 배포판에서 플래그 및 구성 파일을 항상 변경할 수 있는 것은 아니다. 변경 가능한 경우 일반적으로 클러스터 관리자만 변경할 수 있다. 또한 향후 쿠버네티스 버전에서 변경될 수 있으며 이를 설정하려면 프로세스를 다시 시작해야 할 수도 있다. 이러한 이유로 다른 옵션이 없는 경우에만 사용해야 한다.
+
+[ResourceQuota](/ko/docs/concepts/policy/resource-quotas/), [PodSecurityPolicies](/ko/docs/concepts/policy/pod-security-policy/), [NetworkPolicy](/ko/docs/concepts/services-networking/network-policies/) 및 역할 기반 접근 제어([RBAC](/docs/reference/access-authn-authz/rbac/))와 같은 *빌트인 정책 API(Built-in Policy API)* 는 기본적으로 제공되는 쿠버네티스 API이다. API는 일반적으로 호스팅된 쿠버네티스 서비스 및 매니지드 쿠버네티스 설치 환경과 함께 사용된다. 그것들은 선언적이며 파드와 같은 다른 쿠버네티스 리소스와 동일한 규칙을 사용하므로 새로운 클러스터 구성을 반복할 수 있고 애플리케이션과 동일한 방식으로 관리할 수 ​​있다. 또한 안정적인 곳에서는 다른 쿠버네티스 API와 같이 [정의된 지원 정책](/docs/reference/deprecation-policy/)을 사용한다. 이러한 이유로 인해 구성 파일과 플래그보다 선호된다.
```

Comment:
nit; I would write this as:
```suggestion
[ResourceQuota](/ko/docs/concepts/policy/resource-quotas/), [PodSecurityPolicy](/ko/docs/concepts/policy/pod-security-policy/), [NetworkPolicy](/ko/docs/concepts/services-networking/network-policies/) 및 역할 기반 접근 제어([RBAC](/docs/reference/access-authn-authz/rbac/))와 같은 *빌트인 정책 API(built-in policy API)* 는 기본적으로 제공되는 쿠버네티스 API이다. API는 일반적으로 호스팅된 쿠버네티스 서비스 및 매니지드 쿠버네티스 설치 환경과 함께 사용된다. 그것들은 선언적이며 파드와 같은 다른 쿠버네티스 리소스와 동일한 규칙을 사용하므로 새로운 클러스터 구성을 반복할 수 있고 애플리케이션과 동일한 방식으로 관리할 수 ​​있다. 또한 안정적인 곳에서는 다른 쿠버네티스 API와 같이 [정의된 지원 정책](/docs/reference/deprecation-policy/)을 사용한다. 이러한 이유로 인해 구성 파일과 플래그보다 선호된다.
```

changes are to _PodSecurityPolicy_ and _built-in policy API_

Reply:
I agree with you.